examples/Demo/Shared/Components/MarkdownSection.razor.cs

```diff
@@ -21,34 +22,39 @@ public partial class MarkdownSection : FluentComponentBase
     /// Gets or sets the Markdown content 
     /// </summary>
     [Parameter]
-    public string? Content { get; set; }
+    public string? Content
+    {
+        get { return _content; }
+        set
+        {
+            if (_content is string && !_content.Equals(value))
```

Comment:
Why do you test if `_content is string`? `_content` will be always a nullable string. So why not to test `_content is not null`?